src/EnergyPlus/PurchasedAirManager.cc

```diff
@@ -2245,6 +2245,15 @@ namespace PurchasedAirManager {
                     }
                 }
 
+                // If cooling capacity is limited to zero, SupplyMassFlowRate* should be set to zero
```

Comment:
@matthew-larson, @mjwitte I'm wondering if we scale the mass flow rate any time the load is greater than the capacity limit (e.g., `SupplyMassFlowRate = (CoolingLimit/QZnCoolSP) / CpAir / DeltaT`), and not just when the capacity limit is zero. This would modulate the flow rate to supply a consistent deltaT whereas, in the current solution, the DetlaT could drop considerably if the load is greater than the capacity limit.

Reply:
@nealkruis There are already adjustments for capacity limits in place further down in the function. I don't think this is a good reason to change that. That section should have been setting the deltaT to zero so even with some flow it should still have no impact on the zone. But the supply wasn't coming through exactly as the inlet so it reported some load. So this guarantees that the system is off if capacity is zero and there is no OA flow.  

@matthew-larson Did you track down whey the inlet/outlet conditions weren't identical?

Reply:
@mjwitte I found where the zone temperature is modified in the [CorrectZoneAirTemp](https://github.com/NREL/EnergyPlus/blob/9d6116a760ca6317e55196ab157d33f570a42f8b/src/EnergyPlus/ZoneTempPredictorCorrector.cc#L4739) function in ZoneTempPredictorCorrector.cc.  The "Zone Air System Sensible Cooling Rate" is based on `SNLoadCoolRate` which references back to the calculation for [SNLoad](https://github.com/NREL/EnergyPlus/blob/9d6116a760ca6317e55196ab157d33f570a42f8b/src/EnergyPlus/ZoneTempPredictorCorrector.cc#L5059).  The `Node(ZoneNodeNum).Temp` of this calculation is from the corrected zone temperature, but `ZoneEnthalpyIn` traces back to `SumSysMCpT` calculated from the [CalcZoneSums](https://github.com/NREL/EnergyPlus/blob/9d6116a760ca6317e55196ab157d33f570a42f8b/src/EnergyPlus/ZoneTempPredictorCorrector.cc#L4958) function.  This occurs before the corrected zone temperature is set [here](https://github.com/NREL/EnergyPlus/blob/9d6116a760ca6317e55196ab157d33f570a42f8b/src/EnergyPlus/ZoneTempPredictorCorrector.cc#L5049).  I'm not sure if there is a bug here and `SumSysMCpT` or `ZoneEnthalpyIn` should be updated based on the corrected zone temperature or if the zone temperature shouldn't be corrected?  I also could be missing something somewhere in the code.